src/blocks/BulletPointBlockBrandPivot.tsx

```diff
@@ -0,0 +1,167 @@
+import styled from '@emotion/styled'
+import * as React from 'react'
+import {
+  ContentWrapper,
+  getMinimalColorStyles,
+  LAPTOP_BP_UP,
+  MOBILE_BP_DOWN,
+  MOBILE_BP_UP,
+  SectionWrapper,
+  TABLET_BP_UP,
+} from '../components/blockHelpers'
+import { DeferredImage } from '../components/DeferredImage'
+import { getStoryblokImage, Image } from '../utils/storyblok'
+import {
+  BaseBlockProps,
+  MarkdownHtmlComponent,
+  MinimalColorComponent,
+} from './BaseBlockProps'
+
+const BulletPointSectionWrapper = styled(SectionWrapper)({
+  overflowX: 'hidden',
+})
+
+const InnerWrapper = styled('div')({
+  display: 'flex',
+  flexWrap: 'wrap',
+  minWidth: '100%',
+
+  [MOBILE_BP_DOWN]: {
+    paddingLeft: '8px',
+    paddingRight: '8px',
+  },
+
+  [MOBILE_BP_UP]: {
+    marginLeft: '-40px',
+  },
+
+  [LAPTOP_BP_UP]: {
+    marginLeft: '-80px',
```

Comment:
Is there a reason for all the `px` units here?

Reply:
Ah sorry, I should have written a comment about that :) When working with column layouts I like using this margin-left-hack, where you offset the container equally as the margin-left on the items them self. And I also noticed that we couldn't have too much spacing in tablet sizes.

Or did you mean that we should use rems units? @palmenhq

Reply:
I think I'll have a look at building a grid component later on, using css grid with flexbox fallback 😊

Reply:
Yes i was thinking more about whether we should use `rem`s or not really :)

That sounds amazing! Maybe something we could reuse in more places? such as the web onboarding or hedgehog

Reply:
I hear you, I read up a bit on the rem vs px battle today. For font-sizes it's still a no-brainer. But I think we can have look into if we want to use it for all margins as well 👍